sunpy/net/tests/test_attr.py

```diff
@@ -344,11 +345,21 @@ def test_attr_numbes():
 
 
 def test_attr_iterable_length():
+
     # not iterable
-    with pytest.raises(ValueError):
+
+    error_message1 = r"Invalid input value: AIA for key: <class 'sunpy\.net\.tests\.test_attr\.Instrument'>\s*sunpy\.net\.tests\.test_attr\.Instrument\s*Dummy Instrument Class\.\s*Attribute Name\s*Client\s*Full Name\s*Description.*?The value is not iterable or just a string\."
+
+    pattern_1_1 = re.compile(error_message1, re.DOTALL)
+
+    with pytest.raises(ValueError, match=pattern_1_1):
         attr.Attr.update_values({GenericClient: {Instrument: 'AIA'}})
+
     # too many items
-    with pytest.raises(ValueError):
+
+    error_message2 = r"Invalid length \(!=2\) for values: \[\('AIA', 'AIA is Nice', 'Error now'\)\]."
```

Comment:
Just check for "Error now" or "AIA is Nice"

Reply:
Done. Thank You